src/components/core/item-list/components/GridViewSlider.js

```diff
@@ -0,0 +1,57 @@
+// @flow
```

Comment:
Should start with just this component as an isolated PR. And once this is working, replace the one within webapp with this component.